### DISCUSSION TITLE
[IntelliSense] Adding support for Scala.js based libraries

### DISCUSSION THREAD

**Discussion by karan-hsc** [tailwindlabs/tailwindcss#15364] [Ideas]
Some Scala.js based UI libraries ([Laminar](https://laminar.dev/), [Calico](https://www.armanbilge.com/calico/)), use the `cls := ""` syntax for class lists. 

I couldn't figure out how to support that, but if guided I'd be willing to work on a PR

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Reply by karan-hsc** [reply]
That worked, thanks!
Although having it better documented might be helpful for future users.
